mkfs/main.c

```diff
@@ -443,6 +443,7 @@ static const char * const mkfs_usage[] = {
 	OPTLINE("-u|--subvol SUBDIR", "create SUBDIR as subvolume rather than normal directory, can be specified multiple times"),
 	OPTLINE("--shrink", "(with --rootdir) shrink the filled filesystem to minimal size"),
 	OPTLINE("-K|--nodiscard", "do not perform whole device TRIM"),
+	OPTLINE("-z|--compression ALGO", "compression algorithm to use, none (default), zstd"),
```

Comment:
Although it's a nobrainer to choose zstd, mind to add other algos too?

And I'm not sure if David is fine with short options, normally unless it's a very common feature, we only go long options.

Reply:
Yes, I'm planning to add zlib and lzo too.

I can remove `-z` if that's what you want, but it's a very common option to mean "compression" (e.g. tar, rsync)

Reply:
Considering the option only makes sense for `--rootdir`, I'm not too confident about a short option.

And we can always add new short option if it's common utilized later, so far the long option only way looks safer to merge.